rubbery-raccoons/wired/wired_app/models.py

```diff
@@ -0,0 +1,48 @@
+from django.contrib.auth import get_user_model
+from django.db import models
+from django.urls import reverse
+
+
+class Article(models.Model):
+    title = models.CharField(max_length=255)
+    slug = models.SlugField(unique=True)
+    headline = models.TextField()
+    body = models.TextField()
+    publication_date = models.DateField()
+    author = models.ForeignKey(get_user_model(), on_delete=models.SET(None))
+    created = models.DateTimeField(auto_now_add=True)
+    updated = models.DateTimeField(auto_now=True)
+
+    categories = [
+        ("technology", "Technology"),
+        ("business", "Business"),
+        ("culture", "Culture"),
+        ("politics", "Politics"),
+        ("general", "General News"),
+    ]
+    category = models.CharField(max_length=25, choices=categories, default="technology")
+
+    def __str__(self):
+        return f"Article({self.title=}, {self.author=}, {self.publication_date=})"
+
+    def get_absolute_url(self):
+        return reverse("article_detail", kwargs={"slug": self.slug})
+
+    def brief_body(self):
+        if len(self.body) > 20:
+            short = " ".join(self.body.split()[:20])
```

Comment:
I don't like hard-coding "magic" values like this. Use a module-level variable or a class variable, e.g. `if len(self.body) > BODY_BRIEF_MAX_LENGTH`. Or better yet, use a keyword parameter with a default set, e.g. `def brief_body(self, `. Maybe also `get_brief_body` might be a better name for this method.

I'm also a little confused as to how this is supposed to work. If the body has more than 20 _characters_, show the first 20 _words_? I can't tell if this is intended behavior (and a coincidence that you used 20 for both numbers), or a mistake in the code. Using named variables instead of hard-coded "magic" values would make it easier for the reader to understand your intent.